.github/workflows/ROMROM-AUTO-CI-FORMAT.yml

```
@@ -0,0 +1,72 @@
+          git push
+          echo "Auto-formatted code committed"
+
+      - name: Analyze
+        run: |
+          echo "Running Flutter analyze..."
+          flutter analyze
+          echo "Analyze passed"
+
+      - name: Summary
+        if: always()
+        run: |
```

actionlint (1.7.10)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)